rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
+end
```

Comment:
- [ ] Kindly implement the rental adding itself to @book and @person achieving the `belongs_to` association. Here is an example


```ruby
class Rental

attr_accessor :date, :book, :person

  def initialize(date, book, person)
    @book = book
    
    book.rentals << self

    @person = person

    person.rentals << self
  end
end
```